### DISCUSSION TITLE
1980s Echo PC serial port voice synthesizer

### DISCUSSION THREAD

**Discussion by donfroula** [thehighnibble/thehighnibble.github.io#98] [Show and tell]
An early 1980s Echo PC voice synthesizer box with serial port interface. You simply send text to the box over the serial port and it speaks it. I plan on connecting it to my IMSAI 8080 kit when assembled. It should very easy to interface.

The box was a gift from a blind friend, who used it on an early MSDOS PC. It uses the same TI chip as the old "Speak and Spell" toys from the 1980s. I managed to find documentation on the control codes needed to vary a the voice in interesting ways.

Don

https://github.com/thehighnibble/thehighnibble.github.io/assets/9483397/8211bfee-2454-4011-ae1c-f3c9cc0ae405

**Comment by monraet** [comment]
Noticed there is a lot of "Speak and Spell" toys on eBay.  Maybe the TI chip and support chips may be salvageable to make a voice interface for the IMSAI

**Reply by donfroula** [reply]
The chips were amazing, but they needed a host CPU to manage the functions and provide the serial interface. They have a text-to-speech mode, but also a phoneme mode that supports more sophisticated speech generation. The voice is a bit rough! Lowering the pitch emulates the War Games voice a bit better.

I fed it the final speech from "Colossus: The Forbin Project" movie. Kind of chilling!

**Reply by inerlogic** [reply]
of all the emulated devices, clearly there was a mis-step in leaving out the speech synthesizer..... Yes Joshua..... i *would* like to play a game.....
